src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -146,6 +146,14 @@ private function loadExternalFiles(RouteCollection $routeCollection)
     {
         $routeCollection->addCollection($this->fileLoader->load('api.xml'));
 
+        if ($this->container->getParameter('api_platform.enable_entrypoint')) {
```

Comment:
Should not we inject these parameters in the constructor instead?

Reply:
The constructor is already really huge. Also the container is already use at many places. This seems ok.

Reply:
Another option is to use… options, along with the option resolver

Reply:
@greg0ire it's adding a dependency. Which is not good.

Reply:
You can use a simple array, that's possible too